Components/meteoItem.js

```diff
@@ -0,0 +1,14 @@
+ import React from 'react';
+ 
+ import {Text} from 'react-native';
+
+class meteoItem extends React.Component {
+    render() {
+      const meteo=this.props.meteo
+      return (
+        <Text style={{color: 'white'}}>{meteo.city}</Text>        
```

Comment:
For styling, I recccomand using stylesheet instead of inline style

https://reactnative.dev/docs/style